### PR TITLE
Fix Minerva LX info.json

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

Fixes the info.json layout of minerva LX and adds missing column key defs

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/archetype/minervalx/info.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
                 {"matrix": [0, 12], "x": 12, "y": 0},
                 {"matrix": [2, 12], "x": 13, "y": 0},
                 {"matrix": [0, 13], "x": 14, "y": 0},
+                {"matrix": [0, 14], "x": 15, "y": 0},
```
Comment (dunk2k) [inline keyboards/archetype/minervalx/info.json:45]:
```suggestion
                {"matrix": [0, 14], "x": 15.4, "y": 0},
```

File: keyboards/archetype/minervalx/info.json [hunk 2]
```diff
@@ -56,6 +57,7 @@
                 {"matrix": [1, 11], "x": 11.5, "y": 1},
                 {"matrix": [1, 12], "x": 12.5, "y": 1},
                 {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
+                {"matrix": [1, 14], "x": 15, "y": 1},
```
Comment (dunk2k) [inline keyboards/archetype/minervalx/info.json:60]:
```suggestion
                {"matrix": [1, 14], "x": 15.4, "y": 1},
```

File: keyboards/archetype/minervalx/info.json [hunk 3]
```diff
@@ -69,6 +71,7 @@
                 {"matrix": [2, 10], "x": 10.75, "y": 2},
                 {"matrix": [2, 11], "x": 11.75, "y": 2},
                 {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
+                {"matrix": [2, 14], "x": 15, "y": 2},
```
Comment (dunk2k) [inline keyboards/archetype/minervalx/info.json:74]:
```suggestion
                {"matrix": [2, 14], "x": 15.4, "y": 2},
```

File: keyboards/archetype/minervalx/info.json [hunk 4]
```diff
@@ -82,16 +85,18 @@
                 {"matrix": [3, 10], "x": 11.25, "y": 3},
                 {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                 {"matrix": [3, 13], "x": 14, "y": 3},
+                {"matrix": [3, 14], "x": 15, "y": 3},
```
Comment (dunk2k) [inline keyboards/archetype/minervalx/info.json:88]:
```suggestion
                {"matrix": [3, 14], "x": 15.4, "y": 3},
```

File: keyboards/archetype/minervalx/info.json [hunk 5]
```diff
@@ -82,16 +85,18 @@
...
-                {"matrix": [4, 8], "x": 8, "y": 4, "w": 3},
-                {"matrix": [4, 9], "x": 10, "y": 5.25, "w": 1.25},
+                {"matrix": [4, 6], "x": 7, "y": 4, "w": 1.25},
+                {"matrix": [4, 8], "x": 8.25, "y": 4, "w": 1.5},
+                {"matrix": [4, 9], "x": 9.75, "y":4, "w": 1.25},
                 {"matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                 {"matrix": [4, 12], "x": 12.5, "y": 4},
-                {"matrix": [4, 13], "x": 13.5, "y": 4, "w": 1.5}
+                {"matrix": [4, 13], "x": 13.5, "y": 4, "w": 1.5},
+                {"matrix": [4, 14], "x": 15, "y": 4}
```
Comment (dunk2k) [inline keyboards/archetype/minervalx/info.json:99]:
```suggestion
                {"matrix": [4, 14], "x": 15.4, "y": 4}
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/archetype/minervalx/info.json [hunk 6]
```diff
@@ -82,16 +85,18 @@
...
             ]
         }
```
Comment (dunk2k) [inline keyboards/archetype/minervalx/info.json:101]: Upon accepting this suggestion, to conform to PR Checklist can `LAYOUT` macro name, in this file and respective keymaps, be changed to `LAYOUT_all`
```suggestion
        },
        "LAYOUT_ansi": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"matrix": [0, 14], "x": 15.4, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15.4, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15.4, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 1], "x": 2.25, "y": 3},
                {"matrix": [3, 2], "x": 3.25, "y": 3},
                {"matrix": [3, 3], "x": 4.25, "y": 3},
                {"matrix": [3, 4], "x": 5.25, "y": 3},
                {"matrix": [3, 5], "x": 6.25, "y": 3},
                {"matrix": [3, 6], "x": 7.25, "y": 3},
                {"matrix": [3, 7], "x": 8.25, "y": 3},
                {"matrix": [3, 8], "x": 9.25, "y": 3},
                {"matrix": [3, 9], "x": 10.25, "y": 3},
                {"matrix": [3, 10], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 2.75},
                {"matrix": [3, 14], "x": 15.4, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 9], "x": 10, "y":4, "w": 1.25},
                {"matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 12], "x": 12.5, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13.75, "y": 4, "w": 1.25},
                {"matrix": [4, 14], "x": 15.4, "y": 4}
            ]
        },
        "LAYOUT_ansi_split_bs_rshift": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [2, 12], "x": 13, "y": 0},
                {"matrix": [0, 13], "x": 14, "y": 0},
                {"matrix": [0, 14], "x": 15.4, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15.4, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15.4, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 1], "x": 2.25, "y": 3},
                {"matrix": [3, 2], "x": 3.25, "y": 3},
                {"matrix": [3, 3], "x": 4.25, "y": 3},
                {"matrix": [3, 4], "x": 5.25, "y": 3},
                {"matrix": [3, 5], "x": 6.25, "y": 3},
                {"matrix": [3, 6], "x": 7.25, "y": 3},
                {"matrix": [3, 7], "x": 8.25, "y": 3},
                {"matrix": [3, 8], "x": 9.25, "y": 3},
                {"matrix": [3, 9], "x": 10.25, "y": 3},
                {"matrix": [3, 10], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 13], "x": 14, "y": 3},
                {"matrix": [3, 14], "x": 15.4, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 9], "x": 10, "y":4, "w": 1.25},
                {"matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 12], "x": 12.5, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13.75, "y": 4, "w": 1.25},
                {"matrix": [4, 14], "x": 15.4, "y": 4}
            ]
        },
        "LAYOUT_ansi_tsangan": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"matrix": [0, 14], "x": 15.4, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15.4, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15.4, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 1], "x": 2.25, "y": 3},
                {"matrix": [3, 2], "x": 3.25, "y": 3},
                {"matrix": [3, 3], "x": 4.25, "y": 3},
                {"matrix": [3, 4], "x": 5.25, "y": 3},
                {"matrix": [3, 5], "x": 6.25, "y": 3},
                {"matrix": [3, 6], "x": 7.25, "y": 3},
                {"matrix": [3, 7], "x": 8.25, "y": 3},
                {"matrix": [3, 8], "x": 9.25, "y": 3},
                {"matrix": [3, 9], "x": 10.25, "y": 3},
                {"matrix": [3, 10], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 2.75},
                {"matrix": [3, 14], "x": 15.4, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [4, 1], "x": 1.5, "y": 4},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [4, 12], "x": 12.5, "y": 4},
                {"matrix": [4, 13], "x": 13.5, "y": 4, "w": 1.5},
                {"matrix": [4, 14], "x": 15.4, "y": 4}
            ]
        },
        "LAYOUT_ansi_tsangan_split_bs_rshift": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [2, 12], "x": 13, "y": 0},
                {"matrix": [0, 13], "x": 14, "y": 0},
                {"matrix": [0, 14], "x": 15.4, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15.4, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15.4, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 1], "x": 2.25, "y": 3},
                {"matrix": [3, 2], "x": 3.25, "y": 3},
                {"matrix": [3, 3], "x": 4.25, "y": 3},
                {"matrix": [3, 4], "x": 5.25, "y": 3},
                {"matrix": [3, 5], "x": 6.25, "y": 3},
                {"matrix": [3, 6], "x": 7.25, "y": 3},
                {"matrix": [3, 7], "x": 8.25, "y": 3},
                {"matrix": [3, 8], "x": 9.25, "y": 3},
                {"matrix": [3, 9], "x": 10.25, "y": 3},
                {"matrix": [3, 10], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 13], "x": 14, "y": 3},
                {"matrix": [3, 14], "x": 15.4, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [4, 1], "x": 1.5, "y": 4},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [4, 12], "x": 12.5, "y": 4},
                {"matrix": [4, 13], "x": 13.5, "y": 4, "w": 1.5},
                {"matrix": [4, 14], "x": 15.4, "y": 4}
            ]
        },
        "LAYOUT_ansi_wkl": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"matrix": [0, 14], "x": 15.4, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15.4, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15.4, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 1], "x": 2.25, "y": 3},
                {"matrix": [3, 2], "x": 3.25, "y": 3},
                {"matrix": [3, 3], "x": 4.25, "y": 3},
                {"matrix": [3, 4], "x": 5.25, "y": 3},
                {"matrix": [3, 5], "x": 6.25, "y": 3},
                {"matrix": [3, 6], "x": 7.25, "y": 3},
                {"matrix": [3, 7], "x": 8.25, "y": 3},
                {"matrix": [3, 8], "x": 9.25, "y": 3},
                {"matrix": [3, 9], "x": 10.25, "y": 3},
                {"matrix": [3, 10], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 2.75},
                {"matrix": [3, 14], "x": 15.4, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [4, 13], "x": 13.5, "y": 4, "w": 1.5},
                {"matrix": [4, 14], "x": 15.4, "y": 4}
            ]
        },
        "LAYOUT_ansi_wkl_split_bs_rshift": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [2, 12], "x": 13, "y": 0},
                {"matrix": [0, 13], "x": 14, "y": 0},
                {"matrix": [0, 14], "x": 15.4, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15.4, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15.4, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 1], "x": 2.25, "y": 3},
                {"matrix": [3, 2], "x": 3.25, "y": 3},
                {"matrix": [3, 3], "x": 4.25, "y": 3},
                {"matrix": [3, 4], "x": 5.25, "y": 3},
                {"matrix": [3, 5], "x": 6.25, "y": 3},
                {"matrix": [3, 6], "x": 7.25, "y": 3},
                {"matrix": [3, 7], "x": 8.25, "y": 3},
                {"matrix": [3, 8], "x": 9.25, "y": 3},
                {"matrix": [3, 9], "x": 10.25, "y": 3},
                {"matrix": [3, 10], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 13], "x": 14, "y": 3},
                {"matrix": [3, 14], "x": 15.4, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [4, 13], "x": 13.5, "y": 4, "w": 1.5},
                {"matrix": [4, 14], "x": 15.4, "y": 4}
            ]
        }
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]
Looks good pending dunk's suggestions.

**Review by fauxpark — CHANGES_REQUESTED** [review]
Keyboard updates must go through `develop`. Please change the base branch using the Edit button at the top.

**Comment by drashna** [comment]
Looks like there is a merge conflict that needs resolution here.

**Comment by Gondolindrim** [comment]
Tried merging develop but it has too many changes by now. Will try to make a new PR.

**Pull request closed without merging**